myreadinglist/management/commands/stats.py

```diff
@@ -72,5 +74,5 @@ def handle(self, *args, **options):
                          books_completed=books_completed,
                          goals=goals_out)
 
-        for to_email in 'me julian@pybit.es bob@pybit.es'.split():
+        for to_email in PYBITES_EMAIL_GROUP.split():
```

Comment:
no split needed if we change it to `config('PYBITES_EMAIL_GROUP', cast=Csv())` on line 12 (see comment above)